openproblems/tasks/chromatin_potential/README.md

```diff
@@ -0,0 +1,14 @@
+# Chromatin accessibility prediction
+
+Chromatin accessibility prediction refers to the gene expression prediction of a cell or cell type from ATAC-seq peaks. For a summary or all relevant models, see gene score method in [Jeffrey M. Granja et al.](https://www.biorxiv.org/content/10.1101/2020.04.28.066498v1), [Su Wang et al.](https://pubmed.ncbi.nlm.nih.gov/24263090/) et al.
+
+## API
+
+Datasets should contain the following attributes:
+
+* `adata.obsm["gene_score"]` (model-based atac-seq prediction of gene regulation)
+* `adata.obs["atac_rna_cor"]` (predicted correlation between gene expression and atac-seq)
```

Comment:
This section describes what a provided dataset requires, not the output. This should describe the presence of `obsm['mode2'], var.short_gene_name`, etc.

Reply:
Now that I think about this, it would make sense to rename these things -- `adata.X` should be the ATAC-seq, since that's what you're using as input. `adata.obsm['gene_expression']` or something should be the ground truth.

Reply:
Bump.

Reply:
I improve the README based on the latest codes.